### PR TITLE
Add new name "System" and new domain

### PR DESCRIPTION
**Pull request by maya**
**To be merged right after the domain name change**

This adds the new name, shorthand, project domain, GitHub URL, and email address.

Related: https://github.com/18F/web-design-standards/pull/2320

See https://github.com/18F/web-design-standards/issues/2300

[😎 Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/add-name-system/)

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Site settings
-title:  U.S. Web Design Standards
-description: The Standards provide research-backed design patterns for building accessible, responsive, and consistent digital products for the federal government.
+title:  U.S. Web Design System
+description: The Design System provides research-backed design patterns for building accessible, responsive, and consistent digital products for the federal government.
```
Comment (thisisdano) [inline _config.yml]: I wonder if this can be reworked again, to something like:

**Design and build fast, accessible, mobile-friendly government websites backed by user research.**

File: pages/home.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
 ---
 permalink: /
 layout: landing
-title: "U.S. Web Design Standards: A design system for the federal government"
+title: "U.S. Web Design System: A design system for the federal government"
 class: home
 hero:
   callout: A design system for the federal government
-  content: The Standards provide research-backed design patterns for building accessible, responsive, and consistent digital products for the federal government.
+  content: The Design System provides research-backed design patterns for building accessible, responsive, and consistent digital products for the federal government.
```
Comment (thisisdano) [inline pages/home.md]: As above, maybe this can be

**Design and build fast, accessible, mobile-friendly government websites backed by user research.**

File: pages/home.md [hunk 3]
```diff
@@ -58,8 +58,8 @@ We work with you to assess your current system and determine the most cost-effic
     </button>
     <div id="assessment-content" class="usa-accordion-content">
       <ul>
-        <li>We’ll ensure your Standards implementation is in compliance with the official web policy guidance from OMB and your agency.</li>
-        <li>We’ll determine any gaps in skill sets, equipment, design, and code quality needed for successful implementation of the Standards.</li>
+        <li>We’ll ensure your Design System implementation is in compliance with the official web policy guidance from OMB and your agency.</li>
```
Comment (thisisdano) [inline pages/home.md]: `is in compliance with the` s/b `conforms to`

File: pages/getting-started/developers.md [hunk 4]
```diff
@@ -211,16 +211,16 @@ NOTE: If you plan on upgrading to newer versions of the Standards in the future,
...
 
 ## Accessibility
 
-The Standards also meet the [WCAG 2.0 AA accessibility guidelines](https://www.w3.org/TR/WCAG20/) and are compliant with [Section 508 of the Rehabilitation Act](http://www.section508.gov/). We’re happy to answer questions about accessibility — email us for more information.
+The Design System also meets the [WCAG 2.0 AA accessibility guidelines](https://www.w3.org/TR/WCAG20/) and are compliant with [Section 508 of the Rehabilitation Act](http://www.section508.gov/). We’re happy to answer questions about accessibility — email us for more information.
```
Comment (thisisdano) [inline pages/getting-started/developers.md]: `are compliant with` s/b `conforms to`

**Comment by thisisdano** [comment]
Still a little fuzzy on conform/comply, but afaik code conforms, org comply. So, I'm guessing implementations also conform... but 🤷‍♀️

**Comment by maya** [comment]
@thisisdano can you specify a line for the comment https://github.com/uswds/uswds-site/pull/487#issuecomment-359901708 it's a little hard to follow without context?

File: pages/getting-started/developers.md (hunk 4 above)
Comment (maya) [reply pages/getting-started/developers.md]: I put in a PR for the accessibility guide bc their grammar is wrong https://github.com/18F/accessibility/pull/235 https://accessibility.18f.gov/writing-guide/

Comment (maya) [reply pages/getting-started/developers.md]: Fixed in b2f5ad3.

File: pages/home.md (hunk 3 above)
Comment (maya) [reply pages/home.md]: If this is about following the letter of the law, then compliance is still OK.

File: pages/home.md (hunk 2 above)
Comment (maya) [reply pages/home.md]: Updated in 060cb04.

File: _config.yml (hunk 1 above)
Comment (maya) [reply _config.yml]: Updated in 060cb04.

File: pages/home.md (hunk 3 above)
Comment (thisisdano) [reply pages/home.md]: I'm sure either are OK and get the job done, but as I understand it people, agencies, or other intentional agents comply with laws by writing code, producing products, or, in essence producing outputs that conform with the letter of the law. I may be wrong about this, but that's how I've distinguished them in my head!

Comment (thisisdano) [reply pages/home.md]: I can approve this PR, tho :)
